### DISCUSSION TITLE
Matplotlib/Plotly

### DISCUSSION THREAD

**Discussion by Sylvain-BROCAS** [ia-z/ia-z#63] [Polls]
Sur le discord, une discussions a eu lieu à propos du choix de la bibliothèque à utiliser pour tracer les graphiques dans les cours (par exemple dans le chapitre regression linéaire)

2 choix s'offrent à nous:
- Matplotlib:
    - Simple d'utilisation et tout le monde connaît
    - Rapide à mettre en place
- Plotly : 
    - Moins connu que matplotlib mais s'apprend assez facilement : https://github.com/antonin-lfv/Plotly_tutorial
    - Possibilité de faire des diagrammes intéractifs
    - Design globalement plus joli.
    - Code un poil plus long et qui peut faire peur à un débutant
 

Vous pouvez débattre dans la discussion en dessous et voter pour la lib de votre choix. L'objectif est d'homogénéiser les graphes que l'on produira.

**Comment by solomonsanderson** [comment]
I think it is important to note here that Matplotlib and Plotly have slightly different audiences. Matplotlib allows for quick plotting as you mentioned, plots are usually exported as image files or similar.  It is hard to make interactive plots with Matplotlib and requires a large amount of code. 

In comparison, Plotly is much better suited to creating data dashboards. It can be styled with CSS and includes numerous features for making interactive graphs with very little code. It is a more elaborate tool for the more experienced.
